### DISCUSSION TITLE
Allow user to customize the Highest Refresh Rate

### DISCUSSION THREAD

**Discussion by SeasonPhan** [seerge/g-helper#846] [Ideas]
Currently my setup is:
+ Laptop with 120hz screen
+ 144hz External Monitor 
+ Most of the time i turned off my laptop screen and do works on the big monitor

I usually record stuff with OBS. In order to have a smooth recording in 60fps, i have to go to setting and manually change the refresh rate of my External Monitor to 120hz instead of 144hz*(which is automatically detected + set by G-Helper in auto mode)

I would like to have an option to further customize the refresh rate feature of G-Helper according to my need

Would this feature be considered in later versions of G-Helper?

*btw thank you for your work on this awesome tool!

**Comment by seerge** [comment]
@SeasonPhan i have same setup as you do. And I have just manually set my external monitor to 120Hz , so they are in sync and called it a day.

I'm honestly not a big fan in terms of overloading current UI and adding refresh rate switches for each monitor connected / etc as it will cause more issues that benefits :)
